modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Document.js

```diff
@@ -0,0 +1,24 @@
+// Acts a custom Document extension
+import { Node } from '@tiptap/core';
+export default (options) => {
+  const defNode = options.styles.filter(style => style.def)[0];
```

Comment:
Is the styles option mandatory / always set to a fallback value somewhere above this point?

Reply:
Yes, we inherit a styles array from rich-text's `minimumDefaultOptions` if no styles are provided.